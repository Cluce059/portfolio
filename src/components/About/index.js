import React from 'react';
import coverImage  from '../../assets/pfp.jpg';

function About(){
    return (
        <section className="my-5">
            <h1 id="about">Background</h1>
        </section>
    );
}

export default About;