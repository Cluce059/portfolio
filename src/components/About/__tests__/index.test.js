import React from "react";
import { render, cleanup } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import About from '..';

afterEach(cleanup);

describe('About component', () => {
    it('renders', () => {
        render(<About />);
    });
    //double check that matches dom structure as frgament using a snapshot
    it('matches snapshot DOM structure', () => {
        const { asFragment } = render(<About />);
        expect(asFragment()).toMatchSnapshot();
    });
});