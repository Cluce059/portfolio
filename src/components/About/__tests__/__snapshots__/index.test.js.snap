// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About component matches snapshot DOM structure 1`] = `
<DocumentFragment>
  <section
    class="my-5"
  >
    <img
      alt="profilePic"
      class="my-2"
      src="pfp.jpg"
      style="width: 20%;"
    />
    <h1
      id="about"
    >
      Background
    </h1>
    <p>
      Hello, my name is Caroline and I hate this old pic but its the only professional looking one i have rn some space with text until I know what to say here. This could be just a few sentences about my education/certificates and experience so far doing the bootcamp?
    </p>
    <h1>
      Education
    </h1>
    <p>
      UCF Bachelor of Science in Environmental Studies with Geographic Information Systems Certificate
    </p>
    <p>
      current coding bootcamp student
    </p>
    <h1>
      Skills
    </h1>
    <p>
      possibly leave this in resume? 2 much? idk
    </p>
  </section>
</DocumentFragment>
`;
